code/datums/quirks/positive_quirks.dm

```diff
@@ -207,6 +207,85 @@
 	// We've either added or removed TRAIT_NIGHT_VISION before calling this proc. Just refresh the eyes.
 	eyes.refresh()
 
+/datum/quirk/bilingual
+	name = "Bilingual"
+	desc = "Over the years you've picked up an extra language!"
+	icon = FA_ICON_GLOBE
+	value = 4
+	gain_text = span_notice("Some of the words of the people around you certainly aren't common. Good thing you studied for this.")
+	lose_text = span_notice("You seem to have forgotten your second language.")
+	medical_record_text = "Patient speaks multiple languages. Showoff."
+	var/list/possible_languages = list(
+		/datum/language/aphasia,
+		/datum/language/beachbum,
+		/datum/language/calcic,
+		/datum/language/draconic,
+		/datum/language/drone,
+		/datum/language/moffic,
+		/datum/language/monkey,
+		/datum/language/mushroom,
+		/datum/language/nekomimetic,
+		/datum/language/piratespeak,
+		/datum/language/shadowtongue,
+		/datum/language/slime,
+		/datum/language/sylvan,
+		/datum/language/terrum,
+		/datum/language/uncommon,
+		/datum/language/voltaic,
+		/datum/language/xenocommon
+	)
+	var/extra_language
+	mail_goodies = list(/obj/item/taperecorder, /obj/item/clothing/head/frenchberet, /obj/item/clothing/mask/fakemoustache/italian)
+
+/datum/quirk/bilingual/add(client/client_source)
+	var/mob/living/carbon/human/human_holder = quirk_holder
+	//prevents yourself from learning a language you already have
+	for(var/datum/language/spoken as anything in possible_languages)
+		if(human_holder.has_language(spoken))
+			possible_languages -= spoken
+	extra_language = pick(possible_languages)
+	human_holder.grant_language(extra_language, TRUE, TRUE, LANGUAGE_QUIRK)
+
+/datum/quirk/bilingual/remove()
+	var/mob/living/carbon/human/human_holder = quirk_holder
+	human_holder.remove_language(extra_language)
+
+/datum/quirk/mutated
+	name = "Mutated"
+	desc = "You've already had experience as a labrat for the Nanotrasen Genetic Project."
```

Comment:
I don't think I want this

Reply:
Yeah looking at the traits being given I can't see myself ever wanting something like this, we already don't allow stuff like "do more damage", this is *way* more impactful